src/eko/msbar_masses.py

```diff
@@ -143,33 +172,123 @@ def msbar_ker_dispatcher(q2_to, q2m_ref, strong_coupling, fact_to_ren, nf):
     return msbar_ker_exact(a0, a1, order, nf)
 
 
-def evolve_msbar_mass(
+def compute_matching_coeffs_up(nf):
+    """
+    |MSbar| matching coefficients :cite:`Liu:2015fxa` at threshold
+    when moving to a regime with *more* flavors.
+
+    Note :cite:`Liu:2015fxa` (eq 5.) reports the backward relation,
+    multiplied by a factor of 4 (and 4^2 ...)
+
+    Parameters
+    ----------
+        nf:
+            number of active flavors in the lower patch
+
+    Returns
+    -------
+        matching_coeffs_up:
+            downward matching coefficient matrix
+    """
+    matching_coeffs_up = np.zeros((4, 4))
+    matching_coeffs_up[2, 0] = -89.0 / 27.0
+    matching_coeffs_up[2, 1] = 20.0 / 9.0
+    matching_coeffs_up[2, 2] = -4.0 / 3.0
+
+    matching_coeffs_up[3, 0] = -118.248 - 1.58257 * nf
+    matching_coeffs_up[3, 1] = 71.7887 + 7.85185 * nf
+    matching_coeffs_up[3, 2] = -700.0 / 27.0
+    matching_coeffs_up[3, 3] = -232.0 / 27.0 + 16.0 / 27.0 * nf
+
+    return matching_coeffs_up
+
+
+def compute_matching_coeffs_down(nf):
+    """
+    |MSbar| matching coefficients :cite:`Liu:2015fxa` (eq 5) at threshold
+    when moving to a regime with *less* flavors.
+
+    Parameters
+    ----------
+        nf:
+            number of active flavors in the lower patch
+
+    Returns
+    -------
+        matching_coeffs_down:
+            downward matching coefficient matrix
+    """
+    _c = compute_matching_coeffs_up(nf)
```

Comment:
Why `_c`?

`_` have a conventional meaning in python, and in any case this variable has a cryptic name: you're giving up one chance to explain better what you're doing.
If they are coefficients to go up, call them `coeffs_up`.